tank_lab_to_nwb/convert_towers_task/towerspositiondatainterface.py

```diff
@@ -126,13 +114,12 @@ def convert_data(self, nwbfile: NWBFile, metadata_dict: dict, stub_test: bool =
                     pos_timestamps.extend(trial_truncated_time)
                     pos_data = np.concatenate([pos_data, trial_position], axis=0)
 
-            conversion = 1.0  # need to change?
-
             spatial_series_object = SpatialSeries(
-                name='_{}_spatial_series',
-                data=H5DataIO(pos_data, compression='gzip'),
-                reference_frame='unknown', conversion=conversion,
+                name="SpatialSeries",
```

Comment:
Same as above, placeholder name. Sometimes, there are multiple transformations of spatial series belonging to the same Position object (such as normalized or linearized position, etc). I'm assuming we only want this single SpatialSeries per PositionSeries here, since the data didn't indicate anything about transformations.

Reply:
Another common case is multiple body parts tracked at the same time. But correct, here it should just be one